Extension/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Version 0.15.0: February 1, 2018
```

Comment:
Feb 14?

Reply:
Maybe. I wasn't sure if we had other stuff we wanted to ship for February. I wanted to ship earlier so we could get usage data to see if crashes are still an issue or not, particularly with the new IntelliSense compiler.

Reply:
Changed it to the 5th.